### ISSUE TITLE
[FeatureRequest] Telegram support to Operation

### ISSUE THREAD

**Issue by soloam** [mycontroller-org/mycontroller#442]
Hi, it would be nice if MyController supported Telegram as notification, this allows to send all notifications to Telegram and having all information in one place.

Thank You for the great work

**Comment by jkandasa** [comment]
@soloam This feature is implemented in the [SNAPSHOT](https://forum.mycontroller.org/topic/58/download-snapshot-build) version.

Add telegram bot `token` from your bot(`BotFather`) into MyController notifications settings. 
I followed this [link](https://www.youtube.com/watch?v=hJBYojK7DO4) to setup a bot in my telegram account.
![image](https://user-images.githubusercontent.com/1004403/42149953-cdc014c8-7df5-11e8-8351-3eaef7c03e12.png)

Valid `token` will be looking as follows after saved,
![image](https://user-images.githubusercontent.com/1004403/42149979-e42437d0-7df5-11e8-9dee-cb2cd7c1dfc3.png)


Create Operation to send telegram bot message, Here `Channel username` is important.
![image](https://user-images.githubusercontent.com/1004403/42150007-f7aec234-7df5-11e8-91cf-fe2d0aa3a798.png)

**Comment by jl277013** [comment]
I followed the example above.  When I put in the Telegram token, the is bot, Firstname and Username automatically populated fine (FYI, this didn't work in Chrome but was ok in Edge).  I put my Telegram username (is this the same as Channel username?) in the operation as the example above, I got the following error (I substituted my username with "telegram_username" below) from the MyController log:


2018-08-06 16:23:10,192 ERROR [mc-th-pool-6] [org.mycontroller.standalone.operation.TelegramBotUtils:59] Exception: Message(chatId=telegram_username, text=A message from MyController, parseMode=null, disableWebPagePreview=null, disableNotification=null, replyToMessageId=null)
java.lang.RuntimeException: Response(ok=false, result=null, errorCode=400, description=Bad Request: **chat not found**)

Any idea what is the issue?

**Comment by jkandasa** [comment]
@jl277013 can you follow [this video](https://www.youtube.com/watch?v=hJBYojK7DO4)
* join your bot to a group, from 04:20 to 05:52
* to get channel username(chat id), from 09:20 to 13:20.

And I do not see any problem with Chrome browser.

**Comment by jl277013** [comment]
Thanks for the info!  I enabled Allow Groups.  I also enabled Group Privacy.  I was mistakenly used the bot id from the rest api from the description above.  After some digging, I found a page with the following excerpt:

"I dumped the error response and it was 403 "Forbidden: bot can't send messages to bots".
The solution was to go to Telegram and search for the bot called "userinfobot" with the questionmark as an avatar. Click start and it gives you your userid. Use that, and you will start receiving messages from your bot :-)"

When I got the id for my Telegram user, it worked!  
Also, I tried Chrome again today and it seemed to be fine.  I am not sure what happened here.
Thanks again!
